### DISCUSSION TITLE
color shift of the planet when atmosphereEnabled = true

### DISCUSSION THREAD

**Discussion by program-sam** [openglobus/openglobus#807] [Q&A]
Still in love with the atmospheric shader 😍 
![image](https://github.com/openglobus/openglobus/assets/75586344/3669fb8e-f6ce-4949-bde1-bd4c897dfd3e)

But I noticed there is a slight coloration difference. The entire globe seems to get a warmer color tint when enabling atmosphereEnabled:
![image](https://github.com/openglobus/openglobus/assets/75586344/23a0bd95-c6e9-4d08-9af5-00016a896332)

I presume that has something to do with the light source, or the atmospheric effect, but is there a simple way to tweak this? Would be great to still have better color control over all my assets and such. I tried the example in https://sandbox.openglobus.org/examples/lighting but that doesn't seem to have any effect.

This is the difference it makes on my assets for example:
![image](https://github.com/openglobus/openglobus/assets/75586344/256fd7f0-1ff3-47dc-a6ee-c2ef702303ec)

**Comment by Zemledelec** [comment]
Hey @program-sam! Yes sure! 

Check the lighting control plugin and use the values, in layer, and planet variables. I'm sorry, there is no proper API for this, but I have an idea, and even plan to support this, in my next atmosphere tweaking plugin. Which helps to customize atmosphere variables, for Mars for instance...ok here it is:
https://sandbox.openglobus.org/examples/lighting

Enable atmosphere, and try sliders, such as min.opacity/max.opacity.

Where min.opacity means the minimal opacity nearest to the camera, in your case use zero, max opacity - is the farest opacity value, its fine to have 1.7 max.opacity for low altitude, I think.

Cheers!

**Reply by Zemledelec** [reply]
...also, change the base layer, and try diffuse, ambient colors to change.

**Reply by Zemledelec** [reply]
...specular affects for water mask

**Comment by Zemledelec** [comment]
Added the issue! https://github.com/openglobus/openglobus/issues/820

**Comment by Zemledelec** [comment]
I just merged AtmosphereConfig plugin. It's...not user-friendly yet. It should be used with lighting control together.
